src/constants/common.ts

```diff
@@ -106,7 +106,8 @@ export const DEX = 'DEX';
  * Doing any change to the existing values would require creating migration.
  */
 export const STORAGE_KEYS = {
-  mnemonic: 'mnemonic',
+  mnemonic: 'encrypted-mnemonic',
```

Comment:
As far as I understand on mobile we are not using password protection so the mnemonic is not encoded.

Reply:
Mnemonic is always encrypted on mobile from now on